rasa/nlu/classifiers/diet_classifier.py

```diff
@@ -935,7 +935,6 @@ def _entity_label_to_tags(
 
             if self.component_config[BILOU_FLAG]:
                 tags = bilou_utils.ensure_consistent_bilou_tagging(tags)
-                tags = bilou_utils.remove_bilou_prefixes(tags)
```

Comment:
We should also remove this line from the CRF Entity Extractor (https://github.com/RasaHQ/rasa/blob/master/rasa/nlu/extractors/crf_entity_extractor.py#L289).

Reply:
Yes, it was removed from CRF extractor as well. Thanks.